schemer/__init__.py

```diff
@@ -129,6 +125,8 @@ def _verify_type(self, spec, path):
         elif not isinstance(field_type, type) and not isinstance(field_type, types.FunctionType):
             raise SchemaFormatException("Unsupported field type at {}. Type must be a type, a function, an Array or another Schema", path)
 
+    def _valid_schema_default(self, value):
+        return value == {}
```

Comment:
Yes, but they can already do that now within an individual field definition - they can create a default and add validation which contradicts that default. At some point it's the dev's responsibility to not do stupid shit.  

I could see a couple of scenarios where defaults at this level might actually be useful - e.g. you're reusing a common schema for a sub doc but only want to apply a default field at one inclusion point.